iteration/iteration_test.go

```diff
@@ -317,7 +311,7 @@ func (s *TestIterationRepository) TestCreateIterationSameNameFailsWithinSpace()
 	i2 := iteration.Iteration{
 		Name:       name,
 		SpaceID:    space1.ID,
-		UserActive: &userActive,
+		UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c